base/xx-go

```diff
@@ -9,23 +9,45 @@ done
 export GOOS=${TARGETOS}
 export GOARCH=${TARGETARCH}
 
-if [ "$TARGETARCH" = "arm" ]; then
-  if [ -n "$TARGETVARIANT" ]; then
-    case "$TARGETVARIANT" in
-      "v5")
-        export GOARM="5"
-        ;;
-      "v6")
-        export GOARM="6"
-        ;;
-      *)
-        export GOARM="7"
-        ;;
-    esac
-  else
-    export GOARM="7"
-  fi
-fi
+case "$TARGETARCH" in
+  "amd64")
+    if [ -n "$TARGETVARIANT" ]; then
+      case "$TARGETVARIANT" in
+        "v2")
+          export GOAMD64="v2"
+          ;;
+        "v3")
+          export GOAMD64="v3"
+          ;;
+        "v4")
+          export GOAMD64="v4"
+          ;;
+        *)
+          export GOAMD64="v1"
```

Comment:
We should not set it if we don't understand the value. Let `go` manage the default.